keyhive_core/src/keyhive.rs

```diff
@@ -1760,4 +1760,35 @@ mod tests {
 
         assert_eq!(dlg.delegation.subject_id(), doc.borrow().doc_id().into());
     }
+
+    #[tokio::test]
+    async fn cgka_ops_for_doc() {
+        // Create two keyhives, run membership sync (i.e. membersip operatiosn and prekey operations)
+        // then attempt to lookup cgka operations on the other_keyhive for the doc we just synced
+
+        // First create the two keyhives
+        let mut alice = make_keyhive().await;
+        let mut bob = make_keyhive().await;
+
+        // Now make a document on alice which bob can read
+        let bob_indi = Rc::new(RefCell::new(bob.active().borrow().individual().clone()));
+        let doc = alice
+            .generate_doc(vec![bob_indi.dupe().into()], nonempty![[0u8; 32]])
+            .await
+            .unwrap();
+
+        // Now pull out all the membership operations and prekey operations from alice
+        let events = alice.events_for_agent(&bob_indi.into()).unwrap();
+
+        // Now ingest all these events on bob
+        bob.ingest_event_table(events).unwrap(); // Test helper
```

Comment:
`events_for_agent` here is the same as `static_events_for_agent`, but without serialization. We use a helper below that can take unserialized events (though it actually serializes under the hood for code sharing reasons but it's more convenient this way IMO but lemme know if you disagree!)

I also switched to using an ingestion helper. `ingest_event_table` and its sibling `ingest_unsorted_static_events` are hidden behind a `cfg(any(test, feature="test_utils")` because they're not super efficient, but they're really helpful for tests. They run to a fixed point; if the fixed point is empty then you're `Ok`, but `Err` if you don't ingest any new ops between two cycles.